include/solvers/nitsche.h

```diff
@@ -34,53 +34,40 @@ using namespace dealii;
 namespace Parameters
 {
   template <int dim>
-  class Nitsche
+  class NitscheSolid
   {
   public:
-    Nitsche()
+    NitscheSolid()
       : solid_velocity(dim)
     {}
 
     void
-    declare_parameters(ParameterHandler &prm);
+    declare_parameters(ParameterHandler &prm, unsigned int id);
     void
-    parse_parameters(ParameterHandler &prm);
+    parse_parameters(ParameterHandler &prm, unsigned int id);
 
     // Solid mesh
     Parameters::Mesh solid_mesh;
 
-    // Penalization term
-    double beta;
-
     // Solid velocity
     Functions::ParsedFunction<dim> solid_velocity;
     bool                           enable_particles_motion;
 
-    // Calculate forces
-    Verbosity   verbosity;
-    bool        calculate_force_on_solid;
-    bool        calculate_torque_on_solid;
-    Point<dim>  cor; // Center of rotation used for torque calculation
-    std::string force_output_name;
-    std::string torque_output_name;
-
-
-
     // Particle motion integration parameters
     unsigned int particles_sub_iterations;
+
+    // information for force calculation
+    Point<dim> cor; // Center of rotation used for torque calculation
```

Comment:
I find this variable name misleading, as `cor` can also be used for a correction factor (at least this is what came first in mind for be). Maybe use "center" instead, so that it is more transparent for the rest of the code? (I know this variable has not been added in this PR)

Reply:
I'll rename it center_of_rotation it will be clearer